builds/misc/packages-release.yaml

```diff
@@ -435,5 +435,138 @@ stages:
+
+      - bash: |
+          edgeletVersion=$(GitHubParameters.edgeletVersion)
+          iisVersion=$(GitHubParameters.iisVersion)
+
+          wget https://github.com/Azure/azure-iotedge/releases/download/$edgeletVersion/aziot-edge_$edgeletVersion-1_$(os)_$(arch).$(ext) -O $(System.ArtifactsDirectory)/aziot-edge_$edgeletVersion-1_$(os)_$(arch)_github.$(ext)
+          wget https://github.com/Azure/azure-iotedge/releases/download/$edgeletVersion/aziot-identity-service_$iisVersion-1_$(os)_$(arch).$(ext) -O $(System.ArtifactsDirectory)/aziot-identity-service_$iisVersion-1_$(os)_$(arch)_github.$(ext)
+        displayName: Download Edgelet GitHub
+
+      - bash: |
+          # Source the scripts & Update version files
+          sudo chmod +x $(Build.SourcesDirectory)/iotedge/scripts/linux/smokeTestHelper.sh
+          source $(Build.SourcesDirectory)/iotedge/scripts/linux/smokeTestHelper.sh
+
+          if [[ "$(os)" == "ubuntu"* ]]; then
+          # Ubuntu
+            if [[ "$(os)" == *"20.04" ]]; then
+              setup-focal-source-apt
+              setup-config-apt "https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb"
+            elif [[ "$(os)" == *"18.04" ]]; then
+              setup-config-apt "https://packages.microsoft.com/config/ubuntu/18.04/multiarch/packages-microsoft-prod.deb"
+            else
+              echo "Unsupported OS: $(os)"
+              exit 1;
+            fi
+          elif [[ "$(os)" == "debian11" ]]; then
+            setup-config-apt "https://packages.microsoft.com/config/debian/11/packages-microsoft-prod.deb"
+          else
+            echo "Unsupported OS: $(os)"
+            exit 1;
+          fi
+
+          echo $'\n\n================================================\n\n'
+          echo "Aziot-edge"
+          test-released-metadata "aziot-edge" "$(GitHubParameters.edgeletVersion)"
+
+          test-released-artifact \
+            "aziot-edge" \
+            "$(GitHubParameters.edgeletVersion)" \
+            "$(System.ArtifactsDirectory)" \
+            "$(GitHubParameters.edgeletVersion)-1_$(os)_$(arch)_github.$(ext)" \
+            "$(edgelet.maxPercentAllowed)" \
+            "$(IsCheckPreviousPkg)"
+
+          echo $'\n\n================================================\n\n'
+          echo "Aziot-identity-service"
+          test-released-metadata "aziot-identity-service" "$(GitHubParameters.iisVersion)"
+
+          test-released-artifact \
+            "aziot-identity-service" \
+            "$(GitHubParameters.iisVersion)" \
+            "$(System.ArtifactsDirectory)" \
+            "$(GitHubParameters.iisVersion)-1_$(os)_$(arch)_github.$(ext)" \
+            "$(edgelet.maxPercentAllowed)" \
+            "$(IsCheckPreviousPkg)"
+        displayName: Released Artifacts Smoke Tests
+
+      - bash: |
+          # Source the scripts & Update version files
+          sudo chmod +x $(Build.SourcesDirectory)/iotedge/scripts/linux/smokeTestHelper.sh
```

Comment:
Now that you've updated the file mode of smokeTestHelper.sh in this PR, are you going to remove these chmod lines?